CONTRIBUTING.md

```diff
@@ -31,7 +31,7 @@ Once you've downloaded the installer, proceed to the next step to install a proj
 
 Once you have the Eclipse Installer, you can set up both an IDE and a workspace containing the source-code of the project to contribute to. Each Eclipse project provides an individual _Setup_ file, which automates the installation and configuration process. See the [Project Guide](projects.md) for a list of Eclipse IDE projects. 
 
-**💡 Tip:** If you're on a Mac, check out the [additional installation instructions for Mac](InstallerAdvancedOptions.md#install-with-copypaste-for-macos-users).
+**💡 Tip:** If you're on a Mac, you're using the latest version of the installer, because there used to be problems with drag&drop.
```

Comment:
I wonder if we even should keep this little remark to update?
Maybe we should turn it into a general recommendation about using always the latest version and to update preexisting installers if they are reused?

Reply:
Yes. I checked on my Mac right now, and I was in version 1.35 build 812 and it has been automatically updated to build 891 and the d&d is working well now !